README.md

```diff
@@ -1,19 +1,108 @@
-# Volcano dashboard
+# Volcano Dashboard
 
 ## Overview
 
-The volcano dashboard provides a basic dashboard that can be easily deployed in your kubernetes cluster to show the status of resources including volcano jobs, queues, pods, etc.
+The Volcano Dashboard provides a web-based user interface to monitor and manage resources in your Volcano-enabled Kubernetes cluster. This dashboard allows you to visualize and interact with Volcano jobs, queues, pods, and other related resources, making it easier to monitor workload status and troubleshoot issues.
 
-<img src="docs/images/demo.gif" alt="volcano dashboard" style="zoom:50%;" />
+![Volcano Dashboard Demo](docs/images/demo.gif)
 
-## Design
+## Prerequisites
```

Comment:
The `Prerequisites` should exist both in `Installation` and `Development` part.
And the 1, 2, 5 of the Prerequisites should be in `Installation` part, and 3,4 should be in `Development` part.